### PR TITLE
Creating localStorage.pl

### LINKED ISSUE #75: Salvar pontuação máxima localmente (salvar no GameState)

### LINKED ISSUE #76: Recuperar pontuação máxima localmente (salvar no GameState)

### PR DESCRIPTION
**Pull request by edugabriel12**
Creating the localStorage.pl with the respective features:
**saveHighestScore**
**readHighestScore**

### CODE REVIEW

File: Prolog/models/localStorage.pl [hunk 1]
```diff
@@ -1,3 +1,12 @@
...
 
-testMethodLocalStorage(Param):- write(Param).
\ No newline at end of file
+saveHighestScore(Score):- open("score.txt",write,Stream),
+         				  write(Stream,Score),  
+         				  write(Stream,"."), 
+         				  nl(Stream),
+         				  close(Stream).
+
+readHighestScore(Score):- (exists_file("Score.txt") ->  open("score.txt",read,HighestScore),
```
Comment (diego-aquino) [inline Prolog/models/localStorage.pl]: Did you mean `exists_file("score.txt")` (with lowercase "s")?

Comment (edugabriel12) [reply Prolog/models/localStorage.pl]: Yes, my bad.

Comment (edugabriel12) [reply Prolog/models/localStorage.pl]: Already submitted an update changing to lowercase

**Review by diego-aquino — APPROVED** [review]
Good job!

**Pull request merged**